data/src/test/java/com/architectcoders/aacboard/data/repository/RegionRepositoryImplTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.architectcoders.aacboard.data.repository
+
+import com.architectcoders.aacboard.data.datasource.local.AppPermissionChecker
+import com.architectcoders.aacboard.data.datasource.local.AppPermissionChecker.*
+import com.architectcoders.aacboard.data.datasource.local.LocationDataSource
+import com.architectcoders.aacboard.data.repository.RegionRepositoryImpl.Companion.DEFAULT_LANGUAGE
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.*
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.mock
+
+
+@RunWith(MockitoJUnitRunner::class)
+class RegionRepositoryImplTest {
+
+    @Test
+    fun `Returns default value when permission not granted`(): Unit = runBlocking {
```

Comment:
Can't we use `runTest` instead of `runBlocking`?

Reply:
Changed